the_hat_game/players.py

```diff
@@ -59,6 +72,8 @@ def guess(self, words, n_words):
             response_time = response.elapsed.total_seconds()
             response_code = response.status_code
             word_list = response.json()
+            if not validate_word_list(word_list):
+                raise ValidationError("word_list must be a list of strings")
```

Comment:
The same comment as above. See how generic Exception is handled few lines below. Also, there is `if not HIDE_WARNINGS` which serves the purpose of hiding URL of a participant service (because timeout Errors have that URL in the error message). We don't use that info in ValidationError message, so we don't need to use HIDE_WARNINGS.

Reply:
Just to mention this, we'll need to somehow show in logs which player response was incorrect. It seems like we don't have that info here. Maybe we need to unite PlayerDefinition tuple with Player's classes. We need that tuple only for keeping player's `name`. We may as well add that `name` as an argument to `__init__` methods for player classes. We may do that in a separate PR though.

Reply:
fixed in https://github.com/production-ml/the-hat-game/pull/17/commits/86dacda1f8b3bb2f721e0fdc61c9541faecc78bd